bencyn/Questioner-React
# add user registration and logout

#### What does this PR do?

This PR allows a user to register and logout

### Tasks to be completed?
- [x] - implement registration
- [x] - implement logout function

### How can this be manually tested?

- Clone the repo and check out the `user-register` branch

- setup npm
```
$ npm install
```

- run application
```
$ npm run dev
```
